src/main/java/vazkii/patchouli/common/base/PatchouliConfig.java

```diff
@@ -64,6 +65,10 @@ public ClientLoader(ForgeConfigSpec.Builder builder) {
 					.comment("Set this to true to use Shift instead of Ctrl for the inventory quick lookup feature.")
 					.define("quickLookupShift", false);
 
+			overflowMode = builder
+					.comment("Set how text overflow should be coped with: overflow the text off the page, truncate overflowed text, or resize everything to fit.")
+					.defineEnum("textOverflowMode", TextOverflowMode.OVERFLOW);
```

Comment:
Before merge, I'd like to know: Should this be set to OVERFLOW or RESIZE by default?

Reply:
overflow just so it keeps existing behaviour. In a separate PR we could explore setting this on a book or entry specific basis.

Reply:
and we can change the default later once people make use of it for a bit